editors/code/src/main.ts

```diff
@@ -6,25 +6,14 @@ import { type CommandFactory, Ctx, fetchWorkspace } from "./ctx";
 import * as diagnostics from "./diagnostics";
 import { activateTaskProvider } from "./tasks";
 import { setContextValue } from "./util";
-import type { JsonProject } from "./rust_project";
 
 const RUST_PROJECT_CONTEXT_NAME = "inRustProject";
 
-// This API is not stable and may break in between minor releases.
-export interface RustAnalyzerExtensionApi {
```

Comment:
Let's keep the type itself just with the `client` property, that's what we used to have iirc

Reply:
done; kept it.